js/layout-javascript/news-feed-code.js

```diff
@@ -1988,13 +1988,19 @@ DynamicList.prototype.getEntryComments = function(options) {
         return instance.query({
           allowGrouping: true,
           where: {
-            content: content
+            content: content,
+            settings: {
+              text: { $regex: '[^\s]+' }
+            }
           }
         });
       })
-      .then(function(entries){
-        record.comments = entries;
-        record.commentCount = entries.length;
+      .then(function(entries) {
+        var filteredEntries = _.filter(entries, function(entry) {
```

Comment:
Does the regex query work? If so, then we shouldn't need to filter the entries after they have been retrieved.

Reply:
Yeah, it does work. Removed extra filter